config/flatcar/v1_2_exp/translate.go

```diff
@@ -22,9 +22,15 @@ import (
 	"github.com/coreos/vcontext/report"
 )
 
+var (
+	fieldFilters = cutil.NewFilters(types.Config{}, cutil.FilterMap{
```

Comment:
This error says its archutecture related, but its not right? its simply not supported hardware ? no (cex)

Reply:
I've added new error `ErrUnSupportedHardware     = errors.New("the hardware is not supported")`.